src/node_api_types.h

```diff
@@ -67,6 +67,7 @@ typedef enum {
   napi_generic_failure,
   napi_pending_exception,
   napi_cancelled,
+  napi_escape_called_twice,
```

Comment:
Adding a value is okay, inserting it in the middle is not; it changes subsequent values.  You're kind of painted in a corner here because of `napi_status_last`.